vip.py

```diff
@@ -0,0 +1,320 @@
+import requests
+from os.path import join, dirname, exists
+
+
+########################### VARIABLES & ERRORS ################################
+# -----------------------------------------------------------------------------
+__PREFIX = "https://vip.creatis.insa-lyon.fr/rest/"
+__apikey = None
+__headers = {'apikey': __apikey}
```

Comment:
wouln't `__apikey` always be None here ? Would it be better to put `None` instead ?